ocaml/libs/http-svr/http_svr.ml

```diff
@@ -177,7 +179,9 @@ let response_request_header_fields_too_large s =
   in
   response_error_html s "431" "Request Header Fields Too Large" [] body
 
-let response_internal_error ?req ?extra s =
+let response_internal_error ?req ?extra exc s =
+  E.error "Responding with 500 Internal Error due to %s" (Printexc.to_string exc) ;
+  E.log_backtrace () ;
```

Comment:
Can we be sure that we still have the correct backtrace at this point? We had discussion in the past about lost back traces - obviously we will have some backtrace here but is it the correct one? That's why I would like to capture the backtrace first in an exception handler.

Reply:
I'm not sure, but this is a catch-all situation. I guess it's better than nothing at this point.